main.tf

```diff
@@ -0,0 +1,32 @@
+resource "google_storage_bucket" "auto-expire" {
+  name          = var.name
+  project       = var.project
+  location      = "US"
+  force_destroy = true
+
+  storage_class = var.storage_class
+
+  versioning {
+    enabled = var.versioning
+  }
+
+  website {
+    main_page_suffix = "index.html"
+  }
+
+  cors {
+    origin          = "*"
+    method          = "*"
+    max_age_seconds = 3600
+  }
+
+  lifecycle_rule {
```

Comment:
we should make this conditional, so user can enable or disable it.

Reply:
we can have two modules on with and one without this block and then decide which one to use depending on the users inputs